Swiftification.xcodeproj/xcshareddata/xcschemes/Swiftification.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0800"
+   LastUpgradeVersion = "9999"
```

Comment:
Schemes _do_ exist at merely the project level, apparently. (I checked after your previous comment)